plugins/chat/app/serializers/chat/structured_channel_serializer.rb

```diff
@@ -58,11 +58,12 @@ def meta
           chat_message_bus_last_ids[Chat::Publisher::CHANNEL_ARCHIVE_STATUS_MESSAGE_BUS_CHANNEL],
       }
 
-      if id =
-           chat_message_bus_last_ids[
-             Chat::Publisher.user_tracking_state_message_bus_channel(scope.user.id)
-           ]
-        last_ids[:user_tracking_state] = id
+      if !scope.anonymous? &&
+           user_tracking_state_last_id =
+             chat_message_bus_last_ids[
+               Chat::Publisher.user_tracking_state_message_bus_channel(scope.user.id)
+             ]
+        last_ids[:user_tracking_state] = user_tracking_state_last_id
```

Comment:
Oooh anonymous access? Should this be in the serializer?

Also super personal and I know this is a thing.. but assignments in `if` statements are a lil too much of a side effect for my liking.